projects/uitdatabank/toc.json

```diff
@@ -25,28 +25,52 @@
       "uri": "reference/entry.json"
     },
     {
-      "type": "item",
-      "title": "Requirements before going live",
-      "uri": "/docs/entry-api/requirements-before-going-live.md",
-      "slug": "entry-api/requirements-before-going-live"
-    },
-    {
-    "type": "group",
-    "title": "Guides",
-    "items": [
+      "type": "group",
+      "title": "Guides",
+      "items": [
         {
-          "type": "item",
-          "title": "Reusing places & organizers",
-          "uri": "/docs/entry-api/reuse-places-organizers.md",
-          "slug": "entry-api/reuse-places-organizers"
+          "type": "group",
+          "title": "Events",
+          "items": [
+            {
+              "type": "item",
+              "title": "School events",
+              "uri": "/docs/entry-api/events/school-events.md",
+              "slug": "entry-api/events/school-events"
```

Comment:
I will add a redirect in Stoplight to handle this slug change, so the old slug keeps working